src/ui/Nav/AccountInfo.tsx

```diff
@@ -10,7 +10,11 @@ import { getRealmColor } from "shared/constants"
 import Avatar from "shared/components/Media/Avatar"
 import AccountDropdown from "./AccountDropdown"
 
-export default function AccountInfo() {
+type AccountInfoProps = {
+  accountName: string | null
```

Comment:
why can't we just rely on the `const name = useDappSelector(selectWalletName)`? having two sources of truth here smells like a bad idea.